clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -79,6 +100,7 @@ public void clearBufferedDataForUnassignedPartitions(Collection<TopicPartition>
     public synchronized int sendFetches() {
         Map<Node, FetchSessionHandler.FetchRequestData> fetchRequestMap = prepareFetchRequests();
 
+
```

Comment:
extra line 😅

Reply:
Will remove.

Reply:
Removed



the 




extra




line.